server/lib/client.js

```diff
@@ -3,21 +3,22 @@
 const EventEmitter = require('events');
 const lodash = require('lodash');
 
-const RogueEndpointSignups = require('./endpoint-signups');
-const RogueEndpointPosts = require('./endpoint-posts');
+const GatewayEndpointPosts = require('./endpoint-posts');
+const GatewayEndpointSignups = require('./endpoint-signups');
+const GatewayEndpointUsers = require('./endpoint-users');
```

Comment:
This line seems to be pointing to a file that doesn't exist yet.

Reply:
Oh no - this is what happened:
```
Aarons-MBP-2:server aschachter$ git add lib/endpoint-users.js 
The following paths are ignored by one of your .gitignore files:
server/lib/endpoint-users.js
Use -f if you really want to add them.
```